src/Libraries/CoreNodeModels/Remember.cs

```diff
@@ -0,0 +1,148 @@
+using Dynamo.Graph;
+using Dynamo.Graph.Nodes;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using VMDataBridge;
+
+namespace CoreNodeModels
+{
+    [NodeName("Remember")]
+    [NodeDescription(nameof(Properties.Resources.RememberDescription), typeof(Properties.Resources))]
+    [NodeCategory("Core.Data")]
+    [InPortNames(">")]
```

Comment:
??

Reply:
Node is designed to look like a watch node with ">" port strings